### ISSUE TITLE
When copying text, the spaces and newlines are not copied

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#106]
```
What steps will reproduce the problem?
1. Go to the demo page at 
http://flexpaper.googlecode.com/svn/trunk/Example/flash/FlexPaperViewer.html
2. Scroll to page 5
3. Copy the first paragraph and paste to a text editor

What is the expected output? What do you see instead?

Expected:
The £750 million Strategic Investment Fund was created
at Budget 2009 to invest in the UK’s basic capabilities for
industrial innovation, job creation and growth in a highly
competitive global economy

Actual:
The£750millionStrategicInvestmentFundwascreatedatBudget2009toinvestintheUK’sb
asiccapabilitiesforindustrialinnovation,jobcreationandgrowthinahighlycompetitive
globaleconomy

Currently this bug prevent me from using this project, which from any other 
purpose looks great!
```

Original issue reported on code.google.com by `david...@gmail.com` on 16 Aug 2010 at 5:53

**Comment by GoogleCodeExporter** [comment]
```
this was an issue in swftools 0.9.0, please try the latest build (0.9.1) and it 
should work
```

Original comment by `erik.eng...@gmail.com` on 17 Aug 2010 at 6:41

**Comment by GoogleCodeExporter** [comment]
```
there also seems to be an issue in certain newline scenarios where text is not 
returned at all when a newline is included. This will be corrected in 1.4.0 
final
```

Original comment by `erik.eng...@gmail.com` on 17 Aug 2010 at 7:03
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed in 1.4.0 rc2
```

Original comment by `erik.eng...@gmail.com` on 19 Aug 2010 at 5:37
- Changed state: **Fixed**
